slices/rabbitmq-server.yaml

```diff
@@ -0,0 +1,114 @@
+
+  bins:
+    essential:
+      - base-files_base
+      - base-passwd_data
+      - bash_bins
+      - coreutils_bins
+      ## erlang* packages do not exist for i386
+      ## for all other archs, uncomment all those slices below
+      # - erlang-base_bins
+      # - erlang-inets_bins
+      # - erlang-os-mon_bins
+      - grep_bins
+      - libc-bin_locale
+      - libc-bin_nsswitch
+      - libtinfo6_libs
+      - login_bins
+      - netbase_default-hosts
+      - netbase_default-networks
+      - passwd_bins
+      - python3.12_standard
+      - rabbitmq-server_libs
+      - sed_bins
+      - util-linux_bins
+    contents:
+      /usr/bin/rabbitmqadmin:
+      /usr/lib/ocf/resource.d/rabbitmq/rabbitmq-server:
+      /usr/lib/rabbitmq/bin/rabbitmq*:
+      /usr/lib/rabbitmq/bin/vmware-rabbitmq:
+      /usr/lib/rabbitmq/lib/rabbitmq_server-*/escript/rabbitmq-diagnostics:
+      /usr/lib/rabbitmq/lib/rabbitmq_server-*/sbin/rabbitmq*:
+      /usr/lib/rabbitmq/lib/rabbitmq_server-*/sbin/vmware-rabbitmq:
+      /usr/lib/rabbitmq/lib/rabbitmq_server-3.12.1/escript/rabbitmq-plugins:
+        { mutable: true }
+      /usr/lib/rabbitmq/lib/rabbitmq_server-3.12.1/escript/rabbitmq-queues:
+        { mutable: true }
+      /usr/lib/rabbitmq/lib/rabbitmq_server-3.12.1/escript/rabbitmq-streams:
+        { mutable: true }
+      /usr/lib/rabbitmq/lib/rabbitmq_server-3.12.1/escript/rabbitmq-upgrade:
+        { mutable: true }
+      /usr/lib/rabbitmq/lib/rabbitmq_server-3.12.1/escript/rabbitmqctl:
+        { mutable: true }
+      /usr/lib/rabbitmq/lib/rabbitmq_server-3.12.1/escript/vmware-rabbitmq:
+        { mutable: true }
+      /usr/sbin/rabbitmq*:
+      /var/lib/rabbitmq/:
+      /var/lib/rabbitmq/.erlang.cookie: { text: DUMMY, mode: 0400 }
+      /var/lib/rabbitmq/mnesia/: { mode: 0750 }
+      /var/log/rabbitmq/:
+    mutate: |
+      file = content.read(
+        "/usr/lib/rabbitmq/lib/rabbitmq_server-3.12.1/escript/rabbitmq-diagnostics"
+      )
+      content.write(
+        "/usr/lib/rabbitmq/lib/rabbitmq_server-3.12.1/escript/rabbitmq-plugins",
+        file
+      )
+      content.write(
+        "/usr/lib/rabbitmq/lib/rabbitmq_server-3.12.1/escript/rabbitmq-queues",
+        file
+      )
+      content.write(
+        "/usr/lib/rabbitmq/lib/rabbitmq_server-3.12.1/escript/rabbitmq-streams",
+        file
+      )
+      content.write(
+        "/usr/lib/rabbitmq/lib/rabbitmq_server-3.12.1/escript/rabbitmq-upgrade",
+        file
+      )
+      content.write(
+        "/usr/lib/rabbitmq/lib/rabbitmq_server-3.12.1/escript/rabbitmqctl",
+        file
+      )
+      content.write(
+        "/usr/lib/rabbitmq/lib/rabbitmq_server-3.12.1/escript/vmware-rabbitmq",
+        file
+      )
+
+  config:
+    contents:
+      /etc/default/rabbitmq-server:
+      /etc/rabbitmq/rabbitmq-env.conf:
+        { copy: /usr/share/rabbitmq/rabbitmq-env.conf, mode: 0644 }
+
+  libs:
+    essential:
+      ## erlang* packages do not exist for i386
```

Comment:
this is a very important remark, as without these, the SDF won't work. Can you make a remark closer to the top of the file so that it is clear that these dependencies need to be manually added